src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestSuite.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk" InitialTargets="CreateProjectWithColonInSourceName">
```

Comment:
Instead of this, we can add `BeforeTargets="Build"` to `CreateProjectWithColonInSourceName` target.

Reply:
This is the error that I get If I do it:
```
.dotnet/sdk/7.0.100-rc.1.22431.12/NuGet.RestoreEx.targets(19,5): error : (NETCORE_ENGINEERING_TELEMETRY=Restore) One or more errors occurred. (The project file could not be loaded. Could not find a part of the path '/__w/1/s/src/mono/wasm/debugger/tests/debugger-test-with-colon-in-source-name/debugger-test-with-colon-in-source-name.csproj'.  /__w/1/s/src/mono/wasm/debugger/tests/debugger-test-with-colon-in-source-name/debugger-test-with-colon-in-source-name.csproj)
   at Microsoft.Build.Graph.ParallelWorkSet`2.WaitForAllWorkAndComplete()
   at Microsoft.Build.Graph.GraphBuilder.FindGraphNodes()
   at Microsoft.Build.Graph.GraphBuilder.BuildGraph()
   at Microsoft.Build.Graph.ProjectGraph..ctor(IEnumerable`1 entryPoints, ProjectCollection projectCollection, ProjectInstanceFactoryFunc projectInstanceFactory, Int32 degreeOfParallelism, CancellationToken cancellationToken)
   at NuGet.Build.Tasks.Console.MSBuildStaticGraphRestore.LoadProjects(IEnumerable`1 entryProjects)
```